torch/csrc/jit/codegen/cuda/python_frontend/python_bindings.cpp

```diff
@@ -406,260 +422,607 @@ void initNvFuserPythonBindings(PyObject* module) {
+                                                       NvfTensorView*,                        \
+                                                       NvfTensorView*,                        \
+                                                       NvfVal*,                               \
+                                                       NvfVal*,                               \
+                                                       NvfVal*>(                              \
+            {arg1->index, arg2->index, arg3->index, arg4->index},                             \
+            {output->index},                                                                  \
+            static_cast<                                                                      \
+                NvfTensorView* (*)(NvfTensorView*, NvfVal*, NvfVal*, NvfVal*)>(               \
+                torch::jit::fuser::cuda::op_name)));                                          \
+        return output;                                                                        \
+      },                                                                                      \
+      py::return_value_policy::reference);                                                    \
+  nvf_ops.def(                                                                                \
+      op_str,                                                                                 \
+      [](nvfuser::FusionDefinition::Operators& self,                                          \
+         nvfuser::Scalar* arg1,                                                               \
+         nvfuser::Tensor* arg2,                                                               \
+         nvfuser::Scalar* arg3,                                                               \
+         nvfuser::Scalar* arg4) -> nvfuser::Tensor* {                                         \
+        nvfuser::Tensor* output = new nvfuser::Tensor(                                        \
+            self.fusion_definition->recording_state.size());                                  \
+        self.fusion_definition->recording_state.emplace_back(output);                         \
+        self.fusion_definition->recording.emplace_back(new nvfuser::OpRecord<                 \
+                                                       NvfTensorView*,                        \
+                                                       NvfVal*,                               \
+                                                       NvfTensorView*,                        \
+                                                       NvfVal*,                               \
+                                                       NvfVal*>(                              \
+            {arg1->index, arg2->index, arg3->index},                                          \
```

Comment:
arg4 is dropped here? Wow I got really good eyes. :eyes:

Reply:
We don't have clamp and threshold as prims hence untested.

Reply:
Nice catch!  Fixed!